shell/platform/darwin/macos/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -167,4 +199,102 @@ - (bool)testFirstRectForCharacterRange {
   ASSERT_TRUE([[FlutterInputPluginTestObjc alloc] testFirstRectForCharacterRange]);
 }
 
+TEST(FlutterTextInputPluginTest, CanWorkWithFlutterTextField) {
+  FlutterEngine* engine = CreateTestEngine();
+  NSString* fixtures = @(testing::GetFixturesPath());
+  FlutterDartProject* project = [[FlutterDartProject alloc]
+      initWithAssetsPath:fixtures
+             ICUDataPath:[fixtures stringByAppendingString:@"/icudtl.dat"]];
+  FlutterViewController* viewController = [[FlutterViewController alloc] initWithProject:project];
+  [viewController loadView];
+  [engine setViewController:viewController];
+  // Creates a NSWindow so that the native text field can become first responder.
+  NSWindow* window = [[NSWindow alloc] initWithContentRect:NSMakeRect(0, 0, 800, 600)
+                                                 styleMask:NSBorderlessWindowMask
+                                                   backing:NSBackingStoreBuffered
+                                                     defer:NO];
+  window.contentView = viewController.view;
+
+  engine.semanticsEnabled = YES;
+
+  auto bridge = engine.accessibilityBridge.lock();
+  FlutterPlatformNodeDelegateMac delegate(engine, viewController);
+  ui::AXTree tree;
+  ui::AXNode ax_node(&tree, nullptr, 0, 0);
+  ui::AXNodeData node_data;
+  node_data.SetValue("initial text");
+  ax_node.SetData(node_data);
+  delegate.Init(engine.accessibilityBridge, &ax_node);
+  FlutterTextPlatformNode text_platform_node(&delegate, viewController);
+
+  FlutterTextFieldMock* mockTextField =
+      [[FlutterTextFieldMock alloc] initWithPlatformNode:&text_platform_node
+                                             fieldEditor:viewController.textInputPlugin];
+  [viewController.view addSubview:mockTextField];
+  [mockTextField becomeFirstResponder];
+
+  // Sets Client.
```

Comment:
Similar comment; though I'd just delete this one, since it doesn't add much value. Or phrase as:

```cc
// Set the text input client.
```